codecov.yml

```diff
@@ -3,10 +3,10 @@ coverage:
     project:
       default:
         target: auto
-        threshold: 2%
```

Comment:
Global codecov yaml works correctly with "%" and both of the versions pass codecov validation tool.
It looks just a bug of codecov.